substrate/frame/paged-list/src/paged_list.rs

```diff
@@ -135,7 +137,7 @@ where
 	pub fn from_storage() -> Option<Self> {
 		let key = Self::key();
 
-		sp_io::storage::get(&key).and_then(|raw| Self::decode(&mut &raw[..]).ok())
+		storage::get(&key).and_then(|raw| Self::decode(&mut &raw[..]).ok())
```

Comment:
You don't import `sp_io`?

Reply:
> You don't import `sp_io`?

its imported via the testing prelude [here](https://github.com/paritytech/polkadot-sdk/blob/b425c5c8a7fe177bcf554f2b1a1f805f4897638a/substrate/frame/src/lib.rs#L353)

Reply:
ive reverted this and instead i am just importing it via frame::deps since i saw other crates handling sp::io the same way, let me know if thats ok with you